app/views/admin/configs/show.html.erb

```diff
@@ -234,11 +224,59 @@
                                              placeholder: Constants::SiteConfig::DETAILS[:recaptcha_secret_key][:placeholder] %>
                           </div>
                         </div>
+                        <div class="mt-6">
+                          <% if SiteConfig.allow_both_email_signup_and_login %>
+                            <button
+                              class="crayons-btn crayons-btn--danger"
+                              data-action="click->config#activateEmailAuthModal"
+                              <%= SiteConfig.invite_only_mode ? 'disabled' : ''%>>
+                              <% if SiteConfig.invite_only_mode || authentication_enabled_providers.none? %>
+                                <span
+                                  class="crayons"
```

Comment:
```suggestion
                                  class="crayons-tooltip"
```
I think the class you specified here is incorrect, which is why it wasn't rendering :)

Here's what it looks like once you have `crayons-tooltip` as the class:
<img width="858" alt="Screen Shot 2020-10-27 at 4 36 47 PM" src="https://user-images.githubusercontent.com/6921610/97373230-bb247880-1872-11eb-9931-0c3299a88f0a.png">
As you can see, the styling is pretty wonky -- I think you'll need to adjust that so that it renders properly 😅

Reply:
Two options to make it look better. Use whatever you prefer:

1. instead of adding this `span` **inside** button, try to **wrap it** like this:
  ```
  <span class="crayons-tooltip inline-block" data-tooltip="...">
    <button ...>...</button>
  </span>
  ```
2. OR drop the `span` at all, and add tooltip to the `button` like this:
  ```
  <button class="crayons-btn ... crayons-tooltip" data-tooltip="...">...</button>
  ```

Usually I'd go with option 2 and it would work just fine. But because your button is disabled, its opacity is decreased a little (that's the style we apply for disabled buttons). Therefore the tooltip opacity will be decreased as well (since it's attached to that disabled button). And if that bothers you (I personally don't have a problem with that), you can go with option 1 :).